hacking/test-module

```diff
@@ -199,11 +199,9 @@ def runtest(modfile, argspath, modname, module_style, interpreters):
         modfile, argspath = ansiballz_setup(modfile, modname, interpreters)
         if 'ansible_python_interpreter' in interpreters:
             invoke = "%s " % interpreters['ansible_python_interpreter']
-        else:
```

Comment:
This will now break if `module_style` is `ansiballz` and `ansible_python_interpreter` is not set.

Adding `    invoke = ""` at line 198 and removing the else clauses will fix this.